src/System.IO.Pipelines.Networking.Tls/Internal/Sspi/InteropStructs.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace System.IO.Pipelines.Networking.Tls.Internal.Sspi
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct SecPkgInfo
+    {
+        public int fCapabilities;
+        public ushort wVersion;
+        public ushort wRPCID;
+        public int cbMaxToken;
+        public IntPtr Name;
+        public IntPtr Comment;
+    }
+
+    [StructLayout(LayoutKind.Sequential, Pack = 1)]
+    public struct SSPIHandle
+    {
+        public IntPtr handleHi;
+        public IntPtr handleLo;
+
+        public bool IsValid => handleHi != IntPtr.Zero && handleLo != IntPtr.Zero;
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct SecureCredential
+    {
+        public const int CurrentVersion = 0x4;
+        public int version;
```

Comment:
Since CurrentVersion isn't a field, perhaps a newline separator would help readers see that